cmd/cni-metrics-helper/metrics/metrics.go

```diff
@@ -357,7 +361,7 @@ func produceCloudWatchMetrics(t metricsTarget, families map[string]*dto.MetricFa
 					cw.Publish(dataPoint)
 				}
 			case dto.MetricType_SUMMARY:
-				glog.Infof("Produce PERCENTILE metrics: %s, value: %f", action.cwMetricName, action.data.curSingleDataPoint)
+				log.Infof("Produce PERCENTILE metrics: %s, value: %f", action.cwMetricName, action.data.curSingleDataPoint)
```

Comment:
I'm pretty sure the above log messages starting with "Produce..." are not needed and are just fluff.